### PR TITLE
docs(hovercard): DLT-1510 add Hovercard component to doc site

### PR DESCRIPTION
**Pull request by ninamarina**
# Add Hovercard component to doc site

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

- [x] Documentation

## :book: Jira Ticket
[DLT-1510]

## :book: Description
- Adds Hovercard component to the doc site.
- Updates `ComponentVueApi.vue` to consider components that use composition API, and therefore the exported name doesn't have the `dt` prefix (like Datepicker, EmojiPicker, etc).

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :crystal_ball: Next Steps
Complete this documentation, adding helpful variants and examples. For now, we need it to be there since it's available for use.

## :camera: Screenshots / GIFs

image

[DLT-1510]: https://dialpad.atlassian.net/browse/DLT-1510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/dialtone-documentation/docs/components/hovercard.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Hovercard
+description: The Hovercard component provides a popover-like experience for displaying additional information.
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/components/hovercard.md]: "A Hovercard toggles a content overlay when its anchor element is hovered for a minimum amount of time."

File: apps/dialtone-documentation/docs/components/hovercard.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      </div>
+    </div>
+  </div>
+</div>
+'
+vueCode='
+<dt-hovercard placement="bottom-start">
+  <template #anchor>
+    <dt-button kind="muted" importance="outlined">
+      Hover to see the hovercard
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/components/hovercard.md]: "Hover over me for second"

File: apps/dialtone-documentation/docs/components/hovercard.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    </dt-button>
+  </template>
+  <template #content>
+    <div>This is the content of the hovercard</div>
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/components/hovercard.md]: `<div>Content</div>`

File: apps/dialtone-documentation/docs/components/hovercard.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  </template>
+  <template #headerContent>
+    <span>This is the header</span>
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/components/hovercard.md]: `<div>Header</div>`

File: apps/dialtone-documentation/docs/components/hovercard.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  </template>
+  <template #footerContent>
+    <span>This is the footer</span>
```
Comment (francisrupert) [inline apps/dialtone-documentation/docs/components/hovercard.md]: `<div>Footer</div>`

**Review by francisrupert — APPROVED** [review]
Awesome. A few content tweaks. I approve from design perspective, deferring tech review to someone else.

**Review by braddialpad — APPROVED** [review]
Thanks! Looks good codewise. I'd assume design will want to add more written doc at some point.

**Comment by francisrupert** [comment]
> ...assume design will want to add more written doc...

Yep. This is exactly the minimum needed to get it listed here.

File: apps/dialtone-documentation/docs/components/hovercard.md (hunk 2 above)
Comment (ninamarina) [reply apps/dialtone-documentation/docs/components/hovercard.md]: "Hover over me for a second"? Or maybe just "Hover over me"?

File: apps/dialtone-documentation/docs/components/hovercard.md (hunk 3 above)
Comment (ninamarina) [reply apps/dialtone-documentation/docs/components/hovercard.md]: image
Seems like it looks better with more text in it

**Comment by francisrupert** [comment]
eek, slap a `d-wmn332` on it.

image

**Pull request merged**